app/install_packages.sh

```diff
@@ -6,12 +6,14 @@ if [ -z ${PACKAGES} ]; then
 else
   echo 'installing packages'
   apt-get update
+  apt-get install -y ruby2.3 ruby2.3-dev
```

Comment:
If we are not going with using the ruby based docker images we might as well just pass these in via ENV vars so it could be used against multiple versions of ruby. I threw this together quickly.